templates/ra_guac_autoscale_public_alb.template.cfn.yaml

```diff
@@ -432,12 +450,108 @@ Resources:
         configSets:
           config:
             - setup
+            - !If
+              - InstallCloudWatchAgent
+              - cw-agent-install
+              - !Ref "AWS::NoValue"
             - make-guac
             - finalize
           update:
             - setup
             - make-guac
             - finalize
+        cw-agent-install:
+          commands:
+            01-install-cloudwatch-agent:
+              command: !Sub >-
+                aws s3 cp ${CloudWatchAgentUrl} /etc/cfn/scripts/amazon-cloudwatch-agent.rpm &&
+                rpm -v -U /etc/cfn/scripts/amazon-cloudwatch-agent.rpm
+            03-manipulate-cloudwatch-config:
+              command: >-
+                python /etc/cfn/scripts/aws-cw-agent/cloudwatch-applog-config.py
+            10-start-cloudwatch-agent:
+              command: >-
+                /opt/aws/amazon-cloudwatch-agent/bin/amazon-cloudwatch-agent-ctl
+                -a fetch-config -m ec2 -c
+                file:/opt/aws/amazon-cloudwatch-agent/etc/amazon-cloudwatch-agent.json -s
+              cwd: /etc/cfn/scripts/aws-cw-agent
+          files:
+            /opt/aws/amazon-cloudwatch-agent/etc/amazon-cloudwatch-agent.json:
+              content: !Sub |-
+                {
+                  "logs":
+                  {
+                    "logs_collected":
+                    {
+                      "files":
+                      {
+                        "collect_list": [
+                          {
+                            "file_path": "/opt/aws/amazon-cloudwatch-agent/logs/amazon-cloudwatch-agent.log",
+                            "log_group_name": "/aws/ec2/lx/${AWS::StackName}",
+                            "log_stream_name": "cloudwatch_agent_logs_{instance_id}",
```

Comment:
_Just_ updated this log stream name in the wam templates, can you follow the new pattern for log stream names?  `{instance_id}//<path>...` See: https://github.com/plus3it/terraform-aws-watchmaker/pull/144